### ISSUE TITLE
Animation Frame

### ISSUE THREAD

**Issue by iancaffey** [powerbot/RSBot-API#806]
This would be a fantastic hook. Enough of the dynamic sleeping waiting for the animation id to be `-1` for _x_ amount of time when you can check for the id to be `-1` and frame to be `0`.

**Comment by Timer** [comment]
Adding this to back log.

**Comment by Strikeskids** [comment]
Would this be the same as game tick?

**Comment by Timer** [comment]
@Strikeskids no

**Comment by benpoulson** [comment]
@AshGreh Could we add this to the list of future hooks we'd like to add, as I agree it could be a good hook to add. Would be useful to know how far through an animation is for tick-perfect interaction.
I believe runelite refers to this hook as `spotAnimationFrame`. Which can currently be found at:
```
class: by
field: by
multiplier: 1088352163
static: false
```

For now I'm closing this ticket. However, I've made a note to comment on it when this specific hook has been implemented. We have upcoming work to implement a big backlog of new and missing hooks.
